modules/core/src/main/java/org/apache/ignite/internal/commandline/ConnectionAndSslParameters.java

```diff
@@ -17,8 +17,11 @@
 
 package org.apache.ignite.internal.commandline;
 
+import java.util.HashMap;
 import org.apache.ignite.internal.client.GridClientConfiguration;
 
+import java.util.Map;
```

Comment:
Compact import to a single import block.

Reply:
done